### ISSUE TITLE
No ExceptionMapper was found, but must be found.

### ISSUE THREAD

**Issue by theprivateland** [restlet/restlet-framework-java#1070]
I'm experiencing exception in jax-rs mapped resources and I cannot get any output while application complains for no default exception mapper with following message:
"No ExceptionMapper was found, but must be found"

I'ts a bit frustrating and no documentation is to be found online. I know that jax-rs enables you to add custom mappings but for some reason there is not default mapper (??).

How can I add default one for unmapped exceptions? Should there be one? Very confusing. 
Am I missing some jars? Im using 3.0 snaphot.

**Comment by theprivateland** [comment]
I have also experienced another oddity while debugging the problem.
First form parameter in post message is null always (in browser I can see its being sent, its name is "name"). Is there anywhere known issues list for jax-rs with restlet?

This is how I've noticed no default exception mapper.

**Comment by thboileau** [comment]
Hello @theprivateland ,
do you have any sample code?
I've also fix some bugs (cf ticket #1072 ), I wonder if it will help you.

best regards,
Thierry Boileau

**Comment by theprivateland** [comment]
Its just a service throwing exception due to form parameter being null.

It is more flustrating, actually formparam being unset is more important bug than the empty exception.

Application structure:

//jax-rs app
public class JaxRsApp extends JaxRsApplication {
  public JaxRsApp (Context context) {
    super(context);
    this.add(new RestServiceApp());
  }
}

// the app:
public class RestServiceApp extends Application {
    @Override
    public Set<Class<?>> getClasses() {
        Set<Class<?>> resourceClasses = new HashSet<Class<?>>();
        resourceClasses.add(<ClassName>.class);
        return resourceClasses;
    }
}

//and the service:
@POST
  @Produces("application/json") 
  @Path("{clientIdentifier}/pathpart")
  @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
  public ReturnedClass theMethodCalled(
      @PathParam("clientIdentifier") String clientIdentifier,
      @FormParam("name") String theObjectName,
      @CookieParam("ci") String ci) throws SQLException, IOException { /.../

Basically , only FormParam is unset! This is only with POST method. Very frustrating - is there anything wrong? Consumes is added to make exclude data type problems, but doesnt fix anything.

Restlet used is 2.3.2.

**Comment by theprivateland** [comment]
Sorry for late reply, got busy on other things.

**Comment by jlouvel** [comment]
Closing as we are removing JAX-RS in branch 2.5
